recipes/pytorch-dp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pytorch-dp" %}
+{% set version = "0.1b1" %}
+{% set version_long = "0.1-beta.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/{{ name }}/archive/v{{ version_long }}.tar.gz
+  sha256: 77f01cb86183e55c402bdaa05aa52542418c5bea1cfa3f68fabc981c9e1680be
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```

Comment:
This can be noarch: python.  But please remove this line then

Reply:
I've added the noarch: python and removed the skip.